e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -1002,11 +1002,16 @@ H.describeEE("scenarios > embedding > dashboard appearance", () => {
       },
       {
         additionalHashOptions: {
-          locale: "de",
+          locale: "de-CH",
         },
       },
     );
 
+    // We don't have a de-CH.json file, so it should fallback to de.json, see metabase#51039 for more details
+    cy.request("/app/locales/de.json").then(response => {
```

Comment:
Should this be `cy.intercept` instead? `cy.request` makes the request while `cy.intercept` checks if the app makes that request afaik

Reply:
Yes! Thanks for spotting that 🙏

Reply:
106717f8c3715e347ae8c5230597f493ec499aba